### PR TITLE
Phpunit: add tests using phpunit

### PR DESCRIPTION
**Pull request by nicordev**

### CODE REVIEW

File: .testingStuff/doctrine.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?php
```
Comment (olix21) [inline .testingStuff/doctrine.php]: remove `.testingStuff` folder

File: behat.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+default:
```
Comment (olix21) [inline behat.yaml]: could be removed if behat is not used

File: behat.yml.dist [hunk 3]
```diff
@@ -0,0 +1,12 @@
+default:
```
Comment (olix21) [inline behat.yml.dist]: could be removed if behat is not used

**Review by olix21 — APPROVED** [review]
Nice one! :rocket:

File: .testingStuff/doctrine.php (hunk 1 above)
Comment (nicordev) [reply .testingStuff/doctrine.php]: Done.

File: behat.yaml (hunk 2 above)
Comment (nicordev) [reply behat.yaml]: Done.

File: behat.yml.dist (hunk 3 above)
Comment (nicordev) [reply behat.yml.dist]: Not found. Must be already deleted.

**Pull request merged**